pkg/monitortestlibrary/pathologicaleventlibrary/duplicated_event_patterns.go

```diff
@@ -1207,3 +1210,23 @@ func newCrioReloadedTooOftenEventMatcher(finalInternals monitorapi.Intervals) Ev
 		allowIfWithinIntervals: crioReloadedIntervals,
 	}
 }
+
+func newConfigDriftMonitorStoppedTooOftenEventMatcher(finalIntervals monitorapi.Intervals) EventMatcher {
+	configDriftMonitorStoppedIntervals := finalIntervals.Filter(func(eventInterval monitorapi.Interval) bool {
+		return eventInterval.Source == monitorapi.SourceE2ETest &&
+			strings.Contains(eventInterval.Locator.Keys[monitorapi.LocatorE2ETestKey], "imagepolicy signature validation")
```

Comment:
Not all of your tests match this string. Perhaps match on SigstoreImageVerification, and only limit to +/- 30s rather than 10 minutes. This should give a very precise ignore for ConfigDriftMonitorStopped events overlapping with any of your tests.